src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -512,6 +512,12 @@
   - name: --enable-vpa
     type: bool
     short-summary: Enable vertical pod autoscaler for cluster.
+  - name: --nodepool-allowed-host-ports
+    type: string
+    short-summary: Expose host ports on the node pool. When specified, format should be a comma-separated list of ranges with protocol, eg. 80/TCP,443/TCP,4000-5000/TCP.
```

Comment:
If the input is a list, it is a better way to define the parameter with `nargs='+'` and it is not necessary to manually separate the string. After defining with that, the input should be a space-separated list.

Reply:
updated. TIL thanks